From db05b1b83a09bc4b0487be6f7d145907f17c4c12 Mon Sep 17 00:00:00 2001
From: ARATA Mizuki <minorinoki@gmail.com>
Date: Fri, 9 Jan 2015 18:16:02 +0900
Subject: [PATCH] Fix SWIG interface file.

---
 src/bkl_c.i | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/bkl_c.i b/src/bkl_c.i
index 842c8f8..f23950a 100644
--- a/src/bkl_c.i
+++ b/src/bkl_c.i
@@ -42,6 +42,8 @@
         return NULL;
 }
 
+%inline %{
+
 /* Tokenizes input string \a expr that may contain Python expressions
    inside $(...) and calls \a textCallb(\a moreArgs, text)
    for text parts (outside $(...)) and
@@ -92,6 +94,8 @@ extern void proxydict_hijack(PyObject *data, PyObject *dict);
 /* add new dictionary to the proxy: */
 extern void proxydict_add(PyObject *data, PyObject *dict);
 
+%}
+
 %pythoncode %{
 class ProxyDictionary:
     def __init__(self):
